testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/presto/etc/catalog/kafka/schema_with_references.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "type" : "record",
```

Comment:
Is this the correct formatting for this type of file?

Reply:
Checked with other avro specification file in trino. I guess it looks the same